components/transfer/index.en-US.md

```diff
@@ -36,6 +36,7 @@ One or more elements can be selected from either column, one click on the proper
 | style | A custom CSS style used for rendering wrapper element. | object |  |  |
 | targetKeys | A set of keys of elements that are listed on the right column. | string\[] | \[] |  |
 | titles | A set of titles that are sorted from left to right. | ReactNode\[] | - |  |
+| selectAllLabels | A set of customized labels for select all checkboxs on the header | ReactNode[] \| (checked: number, all: number) => ReactNode |  |  |
```

Comment:
* Should be in one object instead of multiple param.
* Transfer use `select` instead `check` as keyword.

```tsx
type selectAllLabels = (ReactNode | (info: { selectedCount: number, totalCount: number }) => ReactNode)[]
```